src/tools/x/src/main.rs

```diff
@@ -28,12 +28,12 @@ fn python() -> &'static str {
 
     for dir in env::split_paths(&val) {
         // `python` should always take precedence over python2 / python3 if it exists
-        if dir.join(PYTHON).with_extension(EXE_EXTENSION).exists() {
+        if dir.join(PYTHON).with_extension(EXE_EXTENSION).try_exists().is_ok() {
```

Comment:
> In case of broken symbolic links this will return Ok(false).

That doesn't seem correct here, does it? Should probably be `if let Ok(true) = ...`

Reply:
The store creates symlinks that do not have targets, but that work through Windows-specific magic trickery and wizardry.
The change you suggest keeps the old behavior. This is why I said that it can be a problem when the python installation is broken, because it doesn't check whether or not the symlink's target exists and is executable (on Unix).
However, it allows to verify that you have access to the symlink itself.

Reply:
Ah, that makes sense (at least on our side, the magic symlinks dont :D).

Could you add a comment to explain this?

Reply:
I added the comment